example/src/pages/form/formikSelect.jsx

```diff
@@ -0,0 +1,142 @@
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      userInput: "",
+      options: [],
+      label: "",
+      disabled: false,
+      error: false,
+      touched: false,
+      field: {
+        name: 'example-input',
+      },
+      serverErrors: {
+        'example-input': null,
+      },
+      form: {
+        errors: { 'example-input': null },
+        touched: { 'example-input': false },
+      }
+    };
+  }
+
+  toggleTouched() {
+    this.setState({
+      touched: !this.state.touched,
+      form: {
+        ...this.state.form,
+        touched: {
+          "example-input": !this.state.form.touched["example-input"]
+        }
+      }
+    });
+  }
+
+  toggleServerError() {
+    this.setState({
+      serverErrors: {
+        'example-input': this.state.serverErrors['example-input'] ? null : 'this input has a server error',
+      }
+    });
+  }
+
+  toggleError() {
+    this.setState({
+      error: !this.state.error,
+      form: {
+        ...this.state.form,
+        errors: {
+          'example-input': this.state.form.errors['example-input'] ? null : 'this input is incorrect',
+        }
+      }
+    });
+  }
+
+  changeDisable() {
+    this.setState({
+      disabled: !this.state.disabled,
+      form: {
+        ...this.state.form,
+        errors: {
+          'example-input': this.state.form.errors['example-input'] ? null : 'this input is incorrect',
+        }
+      }
+    });
+  }
+
+  onChange = e => {
+    const userInput = e.currentTarget.value;
+    this.setState({
+      userInput: e.currentTarget.value
+    });
+  };
+
+  render() {
+    return (
+      <Page>
+        <div>
+          <div>
+            <label htmlFor="isTouched">Touched</label>
+            <input
+              type="checkbox"
+              name="isTouched"
+              value={this.state.touched}
+              onChange={() => this.toggleTouched()}
+            />
+          </div>
+
+          <div>
+            <label htmlFor="hasError">Error</label>
+            <input
+              type="checkbox"
+              name="hasError"
+              value={this.state.error}
+              onChange={() => this.toggleError()}
+            />
+          </div>
+
+          <div>
+            <label htmlFor="hasServerError">has server error?</label>
+            <input
+              type="checkbox"
+              name="hasServerError"
+              onChange={() => this.toggleServerError()}
+            />
+          </div>
+
+          <div>
+            <label htmlFor="isDisabled">Disabled?</label>
+            <input
+              type="checkbox"
+              value={this.state.disabled}
+              name="isDisabled"
+              onChange={() => this.changeDisable()}
+            />
+          </div>
+        </div>
+      ## Dynamic selector
+        <ReactSpecimen span={3}>
+        <FormikSelect
+            label="Gender"
+            options={[{ option: "ONE" },
```

Comment:
[{
label: 'string die getoond word in de dropdown',
value: 1 // waarde die opgeslagen wordt in de state
}]